ui/src/app/components/help-sidebar.tsx

```diff
@@ -111,14 +111,18 @@ const styles = reactStyles({
   },
   runtimeStatusIcon: {
     width: '.5rem',
-    height: '.5rem',
-    margin: '0px .1rem .1rem auto',
-    alignSelf: 'flex-end'
+    height: '.5rem'
   },
   runtimeStatusIconOutline: {
     border: `1px solid ${colors.white}`,
     borderRadius: '.25rem',
   },
+  runtimeStatusIconContainer: {
```

Comment:
You can get rid of the height with the following changes here:

```suggestion
  runtimeStatusIconContainer: {
    alignSelf: 'flex-end',
    margin: '0 .1rem .1rem auto'
  }
```

and then changing the div you added to a FlexRow.

Reply:
ah... much better - thanks!